personal_graph/clients.py

```diff
@@ -58,18 +60,32 @@ def _create_default_client(self):
 
 
 @dataclass
-class OpenAILLMClient(APIClient):
+class LiteLLMClient(APIClient):
     base_url: str = ""
     model_name: str = "openai/gpt-3.5-turbo"
```

Comment:
doesnt seem to be required or used

Reply:
It's basically used when user wants to make use of liteLLM.